demo/index.html

```diff
@@ -63,11 +64,11 @@
         <h4>Basic Menu</h4>
         <div class="horizontal-section">
           <paper-dropdown-menu label="Dinosaurs">
-            <paper-menu class="dropdown-content">
+            <paper-listbox class="dropdown-content">
```

Comment:
I think it may be good for us to take this opportunity to change all of the `paper-menu` elements on this page to `paper-listbox` so that we encourage our users to go that route in the future. WDYT?